l10n_es_aeat_sii_oca/models/account_move.py

```diff
@@ -95,12 +83,12 @@ def _default_sii_refund_type(self):
         "The invoice number should start with LC, QZC, QRC, A01 or A02.",
         copy=False,
     )
-    invoice_jobs_ids = fields.Many2many(
-        comodel_name="queue.job",
-        column1="invoice_id",
-        column2="job_id",
-        relation="account_move_queue_job_rel",
-        string="Connector Jobs",
+    sii_invoice_cron_trigger_ids = fields.Many2many(
```

Comment:
Creo que en lugar de este m2m, ya simplemente debería ponerse un campo con la fecha y hora prevista. Mi idea inicial era que solo existan triggers cuando se le da a enviar inmediatamente, o si en una pasada de la ejecución, todavía quedan facturas pendientes de enviar, pero si no, que se ejecute el cron cíclicamente como estaba previsto. Se puede poner una frecuencia de 5 minutos, o si lo consideráis mucho, 2 o 1 minuto. Si el search a ejecutar es ligero (y se puede hacer que lo sea), no importará demasiado. Cancelar el envío será tan fácil como borrar ese campo de fecha y hora.

Reply:
La idea es realizar un cambio sutil a nivel de jobs vs triggers, donde el usuario/funcionalidades sigan siendo muy parecidas, así no habrá tanto impacto a nivel de usuario.
Si se requiere quitar lo podemos quitar, pero creo que da un uso adecuado.

Reply:
El tema es que esto es más bien técnico y no le aporta al usuario. El usuario necesita saber cuándo se va a enviar. Todo lo demás fueron artificios para ello, y complicamos el código (con los permisos y reglas, los métodos extra en los triggers, etc). Fíjate además en mi comentario sobre no crear triggers si la ejecución no es inmediata.